tools/validators/instance_validator/README.md

```diff
@@ -12,3 +12,13 @@ If you want to use a modified ontology, run the following: `python3 instance_val
 When using a modified ontology, ensure you follow the folder-naming convention: `digitalbuildings/ontology/yaml`. This will allow the instance validator to rely on the new types in the ontology.
 
 Note: as of the current development stage, you must clone the entire repository and run this instance validator script from this directory.
+
+## Using Docker
+
+We provide a Docker solution for clients to run this validator on any platform.
+
+Rename your configuration file to be `config.yaml` and add it to the `tools/validators/instance_validator` folder.
```

Comment:
are we forcing the name?

Reply:
Yes, I can change it to be `building_config.yaml`